go/vt/grpcclient/glogger_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package grpcclient
+
+import (
+	"io"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGlogger(t *testing.T) {
+	gl := glogger{}
+
+	// Redirect stderr to a buffer
+	rescueStderr := os.Stderr
+	r, w, _ := os.Pipe()
+	os.Stderr = w
+
+	gl.Info("temp1")
+	gl.Infof("temp2")
+	gl.Infoln("temp3")
+	gl.Warning("warning")
+
+	w.Close()
```

Comment:
Capture the error and assert that it is nil. Same for all the following occurrences of the same Close function call.

Reply:
Done